docs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "text-detection-api-demo",
+  "private": true,
+  "version": "0.0.0",
+  "description": "Demo for the article about Text Detection API.",
+  "main": "detector.mjs",
+  "scripts": {
+    "start": "http-server ./"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Comandeer/text-detection-api-demo.git"
```

Comment:
All URL-s should be updated to point the new repo.

Reply:
Should I link to the labs repo? :raised\_eyebrow: